src/NiceWrapper.cpp

```diff
@@ -257,11 +282,7 @@ void NiceWrapper::SendLoop() {
   }
 }
 
-void NiceWrapper::SetRemoteCredentials(std::string username, std::string password) {

```

Comment:
Almost correct. ;-) The PC layer still has to extract what it needs (i.e. what is not ICE-specific), libnice then parses the offer/answer/candidate SDPs to get all the ICE-related stuff it needs.

For now, it appears easier/smarter for us to let libnice do its job, but totally open if things need to change.